### PR TITLE
Add support for Django 2.2

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: authority/migrations/0001_initial.py [hunk 1]
```diff
@@ -24,10 +24,10 @@ class Migration(migrations.Migration):
                 ('approved', models.BooleanField(default=False, help_text='Designates whether the permission has been approved and treated as active. Unselect this instead of deleting permissions.', verbose_name='approved')),
                 ('date_requested', models.DateTimeField(default=datetime.datetime.now, verbose_name='date requested')),
                 ('date_approved', models.DateTimeField(null=True, verbose_name='date approved', blank=True)),
-                ('content_type', models.ForeignKey(related_name='row_permissions', to='contenttypes.ContentType')),
-                ('creator', models.ForeignKey(related_name='created_permissions', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
-                ('group', models.ForeignKey(blank=True, to='auth.Group', null=True)),
-                ('user', models.ForeignKey(related_name='granted_permissions', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
+                ('content_type', models.ForeignKey(related_name='row_permissions', to='contenttypes.ContentType', on_delete=models.CASCADE)),
```
Comment (jlward) [inline authority/migrations/0001_initial.py]: Migrations needed to be updated for Django 2.2 (they needed on_delete for all FKs)

File: authority/models.py [hunk 2]
```diff
@@ -16,15 +16,15 @@ class Permission(models.Model):
     of any content type.
     """
     codename = models.CharField(_('codename'), max_length=100)
-    content_type = models.ForeignKey(ContentType, related_name="row_permissions")
+    content_type = models.ForeignKey(ContentType, related_name="row_permissions", on_delete=models.CASCADE)
```
Comment (jlward) [inline authority/models.py]: Django 2.2 requires `on_delete` for all FKs

File: tox.ini [hunk 3]
```diff
@@ -3,7 +3,7 @@ skipsdist = True
 usedevelop = True
 minversion = 1.8
 envlist =
-    py{27,34,35}-dj{18,19,110,111}
```
Comment (jlward) [inline tox.ini]: Dropped support for everything not python2.7/3.7 and django 1.11/2.2

**Comment by jlward** [comment]
For reasons that I am unable to explain, the python3 tests on travis do not seem to be creating the test DB. Not really sure what to do about that.

**Comment by auvipy** [comment]
try to make Travis green

**Comment by jlward** [comment]
> try to make Travis green

I would love to make travis green. However, there seems to be something wonky going on with travis + python3 for this repo. I don't know why, I spent about an hour trying to sort it out. If anyone wants to lend a pair of eyes on it, that would be super. I'm happy to make updates, but I am quickly running out of ideas.

**Comment by jlward** [comment]
```
Tests will fail on SQLite if apps without migrations have relations to apps with migrations. This has been a documented restriction since migrations were added in Django 1.7, but it fails more reliably now. You’ll see tests failing with errors like no such table: <app_label>_<model>. This was observed with several third-party apps that had models in tests without migrations. You must add migrations for such models.
```
This sure looks like what is going on in the python3.7 and django 2.2 tests.

File: example/users/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```
Comment (jlward) [inline example/users/migrations/0001_initial.py]: Had to create initial migration for django 2.2. I added a comment in the PR that was a copy/paste of the line in the django 2.2 docs that gave me the hint for this fix.

**Comment by safwanrahman** [comment]
Looks good. r+ Thanks @jlward. Sorry as it took so long!

**Pull request merged**